### ISSUE TITLE
Custom save-location

### ISSUE THREAD

**Issue by riek-lt** [rybak/resoday#2]
**situationI** 
Often be out of house for over a week, and since the default save location is appdata, I can't sync that up _easily_.
But I do use something like Google Drive and therefore could manage a cross-system save-settings thing.

**Wanted solution**
Not per se other system settings, but definitely the table data and filled-in days be thrown to a file in a location the user can set. 
By doing it this way, I can on the main computer point it to `c:/users/documents/info.db`, and on another computer it imports data from `d:/things/resoday/info.db`, and make it still work.
So settings are different per computer, but actual calendar data is similar.

**Comment by rybak** [comment]
That's a very reasonable request and I would consider implementing this.

There is a workaround: you can pass a path as an argument to Resoday. So far, it was only intended to be used for testing. It is currently _not_ supported as a user-facing feature, so use it on your own risk:
```
java -jar resoday-release.jar <insert path to your directory here>
```
The directory will be used both for data and configuration files.

-----

Here's another much-less-likely-to-work-on-Windows workaround. On Unix-like systems Resoday is compliant with [XDG Base Directory Specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html). Using the Specification, a user on a Unix-like system (e.g. Linux) can override environment variables `XDG_DATA_HOME` and `XDG_CONFIG_HOME` to control where Resoday puts the data and configuration files correspondingly. E.g. via a custom Bash script:

```sh
#!/usr/bin/bash
export XDG_DATA_HOME=...
export XDG_CONFIG_HOME=...
java -jar resoday-release.jar
```

For finding the paths to the directories Resoday uses [library `directories-jvm`](https://github.com/dirs-dev/directories-jvm). The library also supports generating the paths to equivalent directories on Windows.

I don't know it is possible to override the paths generated for Windows in `directories-jvm`. Reading https://github.com/dirs-dev/directories-jvm/issues/49 it seems that it's not possible. Some projects even roll their own implementation just for Windows to make overrides possible https://github.com/sbt/sbt/pull/6408.

**Comment by rybak** [comment]
Preliminary work started in branch https://github.com/rybak/resoday/tree/custom-data-dir

**Comment by rybak** [comment]
Released in v1.3.
